git-sync.sh

```diff
@@ -49,18 +51,28 @@ if [[ -n "$DESTINATION_SSH_PRIVATE_KEY" ]]; then
 fi
 
 # Remove .github directory because we do not want it to be public facing
-ls -la /root/source
-echo "Removing..."
-rm -rf /root/source/.github
-rm -rf /root/source/examples
-
-mv /root/source/README-public.md /root/source/README.md
-
-ls -la /root/source
-git add /root/source/.github
-git add /root/source/examples
-git add /root/source/README.md
-git add /root/source/README-public.md
+echo "Current state of repo"
+ls -la
+
+echo "Replacing public README with README-public"
+mv README-public.md README.md
+rm README-public.md
+
```

Comment:
```suggestion
rm -rf  /.github
rm -rf  /examples
```

Reply:
I think we still need to remove the `.github `and `examples` directory since that is something we do not want students to see